packages/components/src/globals/scss/vendor/@carbon/elements/scss/themes/_tokens.scss

```diff
@@ -96,10 +96,6 @@ $icon-03: map-get($carbon--theme, 'icon-03') !default;
 /// @access public
 /// @group @carbon/themes
 $link-01: map-get($carbon--theme, 'link-01') !default;
-/// @type Color
-/// @access public
-/// @group @carbon/themes
-$inverse-link: map-get($carbon--theme, 'inverse-link') !default;
```

Comment:
Question: I ran `yarn build` in the `components` package to generate vendor files 🤔 Why would the token be deleted? Should I have used a different command?

Reply:
Hmm, I'm not sure. Was it built with latest code? It shouldn't be removing stuff.

Under the hood, components is copy/pasting the elements code over. The dependency version seems to match, as well 🤔 

Have you recently ran:

```bash
yarn clean && yarn install --offline && yarn build
```

And encountered the same issue?

Reply:
Thanks that worked!

I had to run `yarn clean && yarn install --offline && yarn build` first to re-apply these tokens, and THEN run `yarn build` again to regenerate the sass-doc based off the revised generated tokens file 🤔 I think I may have gotten something out of order, but it looks better now & I'll keep this command in mind instead of what I was doing previously 🎉